CodeGeneration/Sources/SyntaxSupport/KeywordSpec.swift

```diff
@@ -0,0 +1,291 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+public struct ContextualKeywordSpec {
```

Comment:
Any reason to have `Contextual` here?

```suggestion
public struct KeywordSpec {
```

Reply:
We already had a `KeywordSpec` in the module but it turns out I could remove it because all contextual keywords are migrated and thus I could now rename `ContextualKeywordSpec` -> `KeywordSpec`.